VERSION

```diff
@@ -1 +1 @@
-0.3.1.20171031.1
+031.20171031.2
```

Comment:
wow, I had no idea meson had this restriction. This is lame. It don't think it really matters since I doubt we are going to update this repo at this point, but with this restriction I can't think of any tagging scheme that isn't ambiguous - if we changed to base tag `0.3.12` the `VERSION` would be `0312.<other_stuff>` and then you can't tell just by looking at the meson-reported version if the base tag is `0.31.2` or `0.3.12` for example